File: OBJCSample/OBJCSample/classes/AZGregorianCalendar.m
//
//  AZGregorianCalendar.m
//  AZCalendarOBJC
//
//  Created by Ali on 4/14/17.
//  Copyright © 2017 Ali Azadeh. All rights reserved.
//

#import "AZGregorianCalendar.h"

@implementation AZGregorianCalendar


+ (NSInteger)getGregorianYearNumber{
    
    NSDateFormatter *yearFormatter;
    yearFormatter = [[NSDateFormatter alloc] init];
    [yearFormatter setDateFormat:@"yyyy"];
    NSString *year = [yearFormatter stringFromDate:[NSDate date]];
    
    return [year integerValue];
    
}
+ (NSInteger)getGregorianMonthNumber{
     NSDateFormatter *monthFormatter;
    monthFormatter = [[NSDateFormatter alloc] init];
    [monthFormatter setDateFormat:@"MM"];
    NSString *month = [monthFormatter stringFromDate:[NSDate date]];
    
    return [month integerValue];
    
}
+ (NSInteger)getGregorianDayNumber{
     NSDateFormatter *dayFormatter;
    dayFormatter = [[NSDateFormatter alloc] init];
    [dayFormatter setDateFormat:@"dd"];
    NSString *day = [dayFormatter stringFromDate:[NSDate date]];
    
    return [day integerValue];
    
}
@end

File: SwiftSample/SwiftSample/Framework/AZCalendarSwift.framework/Headers/AZGregorianCalendar.swift
//
//  AZGregorianCalendar.swift
//  AZCalendarSwift
//
//  Created by Ali on 4/11/17.
//  Copyright © 2017 Ali Azadeh. All rights reserved.
//

import Foundation
public class AZGregorianCalendar {
    
    static func getGregorianYearNumber()->Int{
        let yearFormatter = DateFormatter.init()
        yearFormatter.dateFormat = "yyyy"
        let year = yearFormatter.string(from: Date.init(timeIntervalSinceNow: 0))
        return Int.init(year)!
        
    }
    static func getGregorianMonthNumber()->Int{
        
        let monthFormatter = DateFormatter.init()
        monthFormatter.dateFormat = "MM"
        let month = monthFormatter.string(from: Date.init(timeIntervalSinceNow: 0))
        return Int.init(month)!
        
    }
    
    static func getGregorianDayNumber()->Int{
        let dayFormatter = DateFormatter.init()
        dayFormatter.dateFormat = "dd"
        let day = dayFormatter.string(from: Date.init(timeIntervalSinceNow: 0))
        return Int.init(day)!
        
    }
    
}
